src/components/EventCard.vue

```diff
@@ -26,6 +26,19 @@ export default {
     }
   },
 
-  mixins: ['filters']
+  computed: {
+    eventImage () {
+      console.log(process.env.BASE_URL)
+      return `http://104.131.88.91${this.event.avatar.small}`
```

Comment:
e se a criar um método que retorna isso pra não precisar sempre colocar `http://104.131.88.91` quando for usar? ou algo do tipo

Reply:
Coloquei como variavel de ambiente dentro do quasar.config em build:{ env: {} }